client/components/emojify/style.scss

```diff
@@ -1,10 +1,6 @@
-/**
- * Emojify
- */
-
-// Emoji!!
 .emojify__emoji {
```

Comment:
These are the styles recommended by the twemoji authors:

https://github.com/twitter/twemoji#inline-styles

The old styles left consecutive emoji looking a bit bunched up.